google-http-client-apache/src/main/java/com/google/api/client/http/apache/ApacheHttpTransport.java

```diff
@@ -234,7 +186,13 @@ protected ApacheHttpRequest buildRequest(String method, String url) {
    */
   @Override
   public void shutdown() {
-    httpClient.getConnectionManager().shutdown();
+    if (httpClient instanceof CloseableHttpClient) {
+      try {
+        ((CloseableHttpClient) httpClient).close();
+      } catch (IOException e) {
+        // ignore
```

Comment:
why ignore?

Reply:
Ah, close can throw an `IOException` - I'll let it bubble up.

Reply:
Done.